.github/pull_request_template.md

```diff
@@ -0,0 +1,24 @@
+Thank you for your pull request. Please fill the following information.
+
+---
+
+**Affects User Experience?** Yes/No
+
+If Yes, briefly describe what is affected.
+
+
+**Affects documentation?** Yes/No
```

Comment:
Suggest rephrasing the question, and instead of putting "Yes/No" at the end of the question, can we render these options in a list where one or the other should be selected?  i.e., 

**Will documentation need to be updated based upon changes in this PR?**

- [ ] Yes

- [ ] No